.github/workflows/hil_sample_zephyr.yml

```diff
@@ -160,6 +160,7 @@ jobs:
           git config --global user.name "Git User"
           west patch
 
+          pip3 install allure-pytest
```

Comment:
Is it still needed after adding `allure-pytest` to [pyproject.toml](https://github.com/golioth/golioth-firmware-sdk/pull/594/commits/e1d626ff18e78cef0d8814699258d191a23f4b43#diff-351b54743528ead0e201e113dd12d1392d22392d8b838ea7fe3a8f35ada115c7)?